src/firefly/js/Firefly.js

```diff
@@ -218,12 +218,12 @@ const defFireflyOptions = {
         ...makeDefImageSearchActions(),
     ],
     tap : {
-        services: getTAPServices( ['IRSA', 'NED', 'NASA Exoplanet Archive', 'KOA', 'HEASARC', 'MAST Images',
-                                   'CADC', 'VizieR (CDS)', 'Simbad (CDS)', 'Gaia', 'GAVO', 'HSA', 'NOIR Lab'] ),
+        services: getTAPServicesByName( ['IRSA', 'NED', 'Exoplanet Archive', 'KOA', 'HEASARC', 'MASTImages',
```

Comment:
I see `TapKnownServices: TAP Service: 'Exoplanet Archive' was not found` in console. Extra space?

```suggestion
        services: getTAPServicesByName( ['IRSA', 'NED', 'ExoplanetArchive', 'KOA', 'HEASARC', 'MASTImages',
```

Reply:
I will fix both.